_source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md

```diff
@@ -497,6 +451,6 @@ to OIDC which I strongly recommend you read if you're interested in learning
 more about OIDC. You can [check it out
 here](/blog/2017/07/25/oidc-primer-part-1).
 
-You can also [follow myself](https://twitter.com/rdegges) and
+You can also [follow me](https://twitter.com/rdegges) and
```

Comment:
```suggestion
You can also [follow OktaDev](https://twitter.com/oktadev) and
```
do we need to keep linking to an advocate who's no longer working at Okta? i'm not sure I would... but feel free to disregard this comment...